src/lib389/lib389/password_plugins.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, instance, dn=None):
         self._protected = True
 
 class PBKDF2Plugin(PasswordPlugin):
-    def __init__(self, instance, dn="cn=PBKDF2_SHA256,cn=Password Storage Schemes,cn=plugins,cn=config"):
+    def __init__(self, instance, dn="cn=PBKDF2-SHA256,cn=Password Storage Schemes,cn=plugins,cn=config"):
```

Comment:
Should we change it to PBKDF2-SHA512 here too?

Reply:
I'm not sure why @Firstyear chose SHA256 over SHA512, as this is used by his openldap migration work.  Well I'll change it and see if he yells about it :-)